app/helpers/trails_helper.rb

```diff
@@ -43,7 +43,7 @@ def visit_trail_link(trail)
 
   def auth_to_access_button(video, cta_text: t("trails.start_for_free"))
     cta_classes = "cta-button light-bg"
-    link_to video_auth_to_access_path(video), class: cta_classes do
+    link_to github_auth_path(origin: video_path(video)), class: cta_classes, method: :post do
```

Comment:
this is the important line

Reply:
Does it make sense to have a link with `method: :post`? Maybe we should convert this to a button?

```suggestion
    button_to github_auth_path(origin: video_path(video)), class: cta_classes do
```

Reply:
oh, yeah for sure! Great catch, thanks! Fixed in 14db3e82219263748c8b71824028bc86742067df